packages/frontend/src/components/InfoModal.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './infoModal.css'
+
+const info = [
```

Comment:
I think these are unused.

Reply:
Actually I think this whole file is unused.

Reply:
✅